BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -47,6 +47,7 @@
   private final String folderName = "Download";
   private final String fileName = "library_export_BibBuddy";
   private final String fileTypeBibToast = ".bib";
+
```

Comment:
The folderName, fileTypeBibToast and STORAGE_PERMISSION_CODE always have the same values. Maybe you can create a ExportConstants class for them (e.g. see LibraryKeys class) and also use them in your other export code. Then you could use your constants like this: ExportConstants.STORAGE_PERMISSION_CODE